### ISSUE TITLE
Add buttons for plot navigation with links to all view pages.

### ISSUE THREAD

**Issue by njarboe** [earthref/MagIC#452]
Add buttons for plot navigation with links to all view pages. Otherwise one has to hit the next button too many times.

**Comment by rminnett** [comment]
There aren't any navigation buttons for the list of thumbnails anymore. It loads all of the thumbnails 50 at a time. If the user is looking for the plots for a specific sample, for example, it's better to navigate to that sample in the search interface first and then click on the plot thumbnail to just see the relevant plots.

**Comment by njarboe** [comment]
Many scientists would like to use the plotting feature of MagIC to check on if the data has been uploaded correctly and has any obvious problems. Being able to scroll through all the plots quickly and glance at the tumbnails would be great. Click on a button to load all the Equal Area plots, go do something else while they load, and then come back to look at them when they are ready. Maybe this feature would be too intensive a use for our computer resources, but it would be very nice to have.

**Comment by rminnett** [comment]
> " Click on a button to load all the Equal Area plots, go do something
else while they load, and then come back to look at them when they are
ready."

Can't you do that now by clicking on the plot thumbnail for the
contribution, selecting "Equal Area", and waiting for them all to load so
you can scroll through them?

**Comment by njarboe** [comment]
The plotting gets hung up on my computer. Sometimes the plots are all finished loading on the screen but no new "loading" plots show up for over five minutes. New plots take longer and longer to show up for some reason and so on the big ones it looks like they just stop loading, although the three dots are still at the end (that is meant to signal plots are still loading, correct? The three dots to me are not intuitively a "still loading" icon. I think text stating "wait for more plots" would be better, but if the loading issue can be fixed that change is not really necessary). I eventually waited over five minutes for the next set of plots to start loading. It loaded the next set fairly quickly, but then hung up again. The website was also unreachable when I was trying this. 

I am looking at the Equal Area plots on the Meijer et al. (2019) contribution (it's on the front page).

**Comment by rminnett** [comment]
Yes, trying to view 1k+ thumbnails on one page will crash the browser because it can't handle rendering that much content - it's doesn't affect the server or website for other users. I'll see if I can let the thumbnails continue to load with scrolling whilst only rendering what's currently visible (similarly to how the search results are displayed).

**Comment by njarboe** [comment]
Ok. Maybe my computer is just getting a bit old and is not fast enough, but I imagine that others may be in a similar situation, especially PhD students.
